### DISCUSSION TITLE
Where did the option to choose address books go?

### DISCUSSION THREAD

**Discussion by annettekusma** [FossifyOrg/General-Discussion#682] [Q&A] [answered]
When the app is freshly installed, the search bar has those three symbols in it: a sort of list symbol, a filter symbol, and three dots. After I clicked on a  few contacts, the search bar had other three symbols in it: 3x3 squares, gear symbol and information symbol. Why does it change? Where did the option to choose address books go (the filter symbol)? Why is that option not part of the settings?

**Comment by Aga-C** [comment, marked as answer]
It will be fixed in the next version: https://github.com/FossifyOrg/General-Discussion/issues/156.
